src/test/regress/expected/multi_partitioning.out

```diff
@@ -923,15 +923,340 @@ ERROR:  no partition of relation "multi_column_partitioning_1660068" found for r
+BEGIN;
+SELECT master_modify_multiple_shards('UPDATE partitioning_locks SET time = ''2009-03-01''');
+ master_modify_multiple_shards 
+-------------------------------
+                             0
+(1 row)
+
+SELECT relation::regclass, locktype, mode FROM pg_locks WHERE relation::regclass::text LIKE 'partitioning_locks%' ORDER BY 1, 2, 3;
+        relation         | locktype |       mode       
+-------------------------+----------+------------------
+ partitioning_locks      | relation | AccessShareLock
+ partitioning_locks      | relation | RowExclusiveLock
+ partitioning_locks_2009 | relation | RowExclusiveLock
+ partitioning_locks_2010 | relation | RowExclusiveLock
 (4 rows)
 
-DROP TABLE IF EXISTS partitioning_test_2012, partitioning_test_2013, partitioned_events_table, partitioned_users_table, list_partitioned_events_table, multi_column_partitioning;
+COMMIT;
+-- test locks on DDL
+BEGIN;
+ALTER TABLE partitioning_locks ADD COLUMN new_column int;
+SELECT relation::regclass, locktype, mode FROM pg_locks WHERE relation::regclass::text LIKE 'partitioning_locks%' ORDER BY 1, 2, 3;
+        relation         | locktype |        mode         
+-------------------------+----------+---------------------
+ partitioning_locks      | relation | AccessExclusiveLock
+ partitioning_locks      | relation | AccessShareLock
+ partitioning_locks_2009 | relation | AccessExclusiveLock
+ partitioning_locks_2009 | relation | RowExclusiveLock
+ partitioning_locks_2010 | relation | AccessExclusiveLock
+ partitioning_locks_2010 | relation | RowExclusiveLock
+(6 rows)
+
+COMMIT;
+-- test locks on TRUNCATE
+BEGIN;
+TRUNCATE partitioning_locks;
+SELECT relation::regclass, locktype, mode FROM pg_locks WHERE relation::regclass::text LIKE 'partitioning_locks%' ORDER BY 1, 2, 3;
+        relation         | locktype |        mode         
+-------------------------+----------+---------------------
+ partitioning_locks      | relation | AccessExclusiveLock
+ partitioning_locks      | relation | AccessShareLock
+ partitioning_locks_2009 | relation | AccessExclusiveLock
+ partitioning_locks_2009 | relation | AccessShareLock
+ partitioning_locks_2009 | relation | RowExclusiveLock
+ partitioning_locks_2009 | relation | ShareLock
+ partitioning_locks_2010 | relation | AccessExclusiveLock
+ partitioning_locks_2010 | relation | AccessShareLock
+ partitioning_locks_2010 | relation | RowExclusiveLock
+ partitioning_locks_2010 | relation | ShareLock
+(10 rows)
+
+COMMIT;
+-- test shard resource locks with master_modify_multiple_shards
+BEGIN;
+SELECT master_modify_multiple_shards('UPDATE partitioning_locks_2009 SET time = ''2009-03-01''');
+ master_modify_multiple_shards 
+-------------------------------
+                             0
+(1 row)
+
+-- see the locks on parent table
+SELECT
+    locktype,
+    mode
+FROM
+    pg_locks AS l JOIN pg_dist_shard AS s
+ON
+    l.objid = s.shardid
+WHERE
+    logicalrelid = 'partitioning_locks'::regclass
```

Comment:
It'd be nice to see the locks on the `partitioning_locks_2009 ` and `partitioning_locks_2010`.


Could you add the `logicalrelid`::regclass to the output and add `WHERE logicalrelid IN (`..` , '...')` to see those? And remove the query below?